config.yml

```diff
@@ -43,12 +49,57 @@ apps:
     otpe:
         image: "804282218731.dkr.ecr.us-west-2.amazonaws.com/ocr-telemetry-prometheus-exporter"
         version: "2496ee62ed6c81898ddcc42d40d3be1f9c7ac549"
+        requests:
+            memory: 800m
+            cpu: 1
+        limits:
+            memory: 1.4G
+            cpu: 1.3
     explorer:
         image: "studentcuza/explorer"
         version: "choose-security-protocol"
+        requests:
+            memory: 800M
+            cpu: 1
+        limits:
+            memory: 1G
+            cpu: 1.5
     atlas-evm:
         image: "804282218731.dkr.ecr.us-west-2.amazonaws.com/atlas-evm"
         version: "2931b7e2c13ad08742a9d825c915cc01663de5c0"
+        requests:
+            memory: 200M
+            cpu: 0.25
+        limits:
+            memory: 350m
+            cpu: 0.4
+    kafka-rest:
+        image: confluentinc/cp-kafka-rest
+        version: 7.0.0
+        requests:
+            memory: 150m
```

Comment:
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes
You can express memory as a plain integer or as a fixed-point number using one of these suffixes: E, P, T, G, M, k. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki

So thus these lowercase 'm' should be invalid.

Reply:
I guess it doesn't care about capitalization, I deployed with this code and it worked